packages/addons/addon-depends/network-tools-depends/iftop/package.mk

```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="iftop"
-PKG_VERSION="1.0pre4"
-PKG_SHA256="f733eeea371a7577f8fe353d86dd88d16f5b2a2e702bd96f5ffb2c197d9b4f97"
+PKG_VERSION="77901c8c53e01359d83b8090aacfe62214658183"
```

Comment:
is the latest git version needed to make it work?

Reply:
No. I did review Paul’s changes and believe they have merit - eg 1Gbps/DNS

- I have reached out to Paul with our patches too.
- I did consider (and look at incorporating some of his patches) but the resulting patch was huge ...